src/odemis/gui/cont/acquisition.py

```diff
@@ -714,6 +714,18 @@ def on_acquisition(self, evt):
         Start the acquisition (really)
         Similar to win.acquisition.on_acquire()
         """
+        # Time-resolved data cannot be saved in .ome.tiff format for now
+        for s in self._tab_data_model.streams.value:
+            if (isinstance(s, ScannedTemporalSettingsStream) or
+                isinstance(s, ScannedTCSettingsStream) or
+                isinstance(s, TemporalSpectrumSettingsStream)):
+                has_temporal = True
+            else:
+                has_temporal = False
+        if self.conf.last_format != u'HDF5' and has_temporal:
```

Comment:
Let's make it more explicit: if the format is "TIFF". Also, please add a comment explaining that OME-TIFF wants to save each time data on a separate "page", which causes too many pages.

Reply:
done